README.md

```diff
@@ -33,6 +33,24 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
 
 # About DevTools
 
+## Highlights
+
+* [Inactive CSS](https://hacks.mozilla.org/2019/10/firefox-70-a-bountiful-release-for-all/#developertools):
+Firefox DevTools now grays out CSS declarations that don’t have an effect on the page. When you hover over the info icon, you’ll see a useful message about why the CSS is not being applied, including a hint about how to fix the problem.
+![Inactive_CSS](https://www.mozilla.org/media/img/firefox/developer/hero-inactive-css.40a2e0b8ddb1.png) 
```

Comment:
Let's remove all the images for now, until we know what the design will be with the other work that's happening.